index.html

```diff
@@ -0,0 +1,136 @@
+<!doctype html>
+<html>
+  <head>
+    <title> Aloha Apparel Co</title>
+    <link rel="stylesheet" type="text/css" href="css/normalize.css">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+    <script src="https://use.fontawesome.com/90f704d235.js"></script>
+    <script src="lib/flickity/flickity.min.css"></script>
+  </head>
+
+  <body>
+    <header>
+      <div class="logo">
+        <img src="images/aloha-logo.svg"> 
+      </div>
+          
+      <div class="nav">
+        <a href="google.com">ABOUT</a>
+        <a href="google.com">SHOP</a>
+        <a href="google.com">FEATURED</a>
+        <a href="google.com">UPDATES</a>
+      </div>
+            
+      <div class="shoppingcart">
+        <a href="google.com"> <img src="images/cart-icon.svg"></a>
+      </div>
+
+    </header>
+
+<div class="hero-banner">
+    <p>NEW COLLECTION</p>
```

Comment:
Don't use all caps letters for style reasons. Use `text-transform: uppercase` in your CSS instead.